content/md/imports-exports/quick-export.md

```diff
@@ -41,6 +41,8 @@ The created files enclose one line per product or product model exported, and a
 
 Only the working channel is taken into account, and thus all the locale information required for the channel is exported. The working locale is not taken into account.
 
+Quick export file of the context grid will not include “read only” values, such as completeness, created at, updated at, variant products, etc. The label is not exported, as it is considered as a property and group column is exported by default as empty.
```

Comment:
It's not only for the context grid export, completeness, created at, updated at, variant products are never exported in all exports. 
I will remove the group column because it's not exported in the case of the grid context export if the group column is not displayed in the grid.

Reply:
We should use a 
::: warning format, to highlight it.
Maybe we should a simple text like: 
::: warning
Note that the following columns are not exported during a Grid Context Export: Labels, Completeness, Created at, Updated at, Variant products (we need to list them all). 
:::